BundID/Views/SupplementaryViews/DialogButtons.swift

```diff
@@ -9,7 +9,17 @@ struct DialogButtons<Action>: View {
     
     struct ButtonConfiguration {
         let title: String
-        let action: Action
+        let actionCallback: () -> Action
+        
+        init(title: String, action: Action) {
+            self.title = title
+            self.actionCallback = { action }
+        }
+        
+        init(title: String, actionCallback: @escaping () -> Action) {
```

Comment:
We could have used an `@autoclosure` here to get rid of the other initializer, right? And then we could still use the `action` label instead of `actionCallback`.